ref/gl/gl_image.c

```diff
@@ -1070,7 +1070,7 @@ static void GL_TextureImageRAW( gl_texture_t *tex, GLint side, GLint level, GLin
 	}
 	else if( tex->target == GL_TEXTURE_2D_MULTISAMPLE )
 	{
-#if !defined( XASH_GLES ) && !defined( XASH_GL4ES )
+#if !defined( XASH_GL_STATIC ) || (!defined( XASH_GLES ) && !defined( XASH_GL4ES ))
```

Comment:
What's the point for checking for XASH_GL_STATIC here? It doesn't seem to change anything for MSAA.

Reply:
If you're updating macros, don't forget to update comments after #else and #endif